gym_minigrid/minigrid.py

```diff
@@ -621,8 +621,8 @@ class MiniGridEnv(gym.Env):
     """
 
     metadata = {
-        'render.modes': ['human', 'rgb_array'],
-        'video.frames_per_second' : 10
+        'render_modes': ['human', 'rgb_array'],
```

Comment:
For backward compatibility, could you keep the old values as well with a comment